CMakeLists.txt

```diff
@@ -23,18 +23,13 @@ if(${CMAKE_CURRENT_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_BINARY_DIR})
 	message(FATAL_ERROR "Prevented in-tree build.")
 endif()
 
-cmake_minimum_required(VERSION 3.7)
+cmake_minimum_required(VERSION 3.18)

```

Comment:
I think CMake 3.18 is bit too high but I don't think it's an issue for us. It could be an issue with downstream though.